pyiron/vasp/base.py

```diff
@@ -942,29 +942,10 @@ def calc_minimize(
             self.write_charge_density = retain_charge_density
         if retain_electrostatic_potential:
             self.write_electrostatic_potential = retain_electrostatic_potential
-        if ionic_forces is not None:
```

Comment:
I guess this part should actually stay, because `ionic_forces` should be replaced by `ionic_force_tolerance` (even though personally I don't really care...)

Reply:
Sure but these lines are there in `set_convergence_precision`

Reply:
So from what I understood in a recent discussion, we should replace all `ionic_forces` by `ionic_force_tolerance`. I guess it's just I missed them in `set_convergence_precision` when I did the conversion last time. But maybe @liamhuber can say more about it, because I'm personally fine with `ionic_forces` as well.